tensorflow/core/kernels/mlir_generated/build_defs.bzl

```diff
@@ -89,16 +89,16 @@ _gen_mlir_op_rule = rule(
     implementation = _gen_mlir_op_impl,
 )
 
-def _gen_mlir_op(op, type, platform, output_type):
+def _gen_mlir_op(op, name, type, platform, output_type):
     _gen_mlir_op_rule(
-        name = "generate_{op}_{platform}_{type}_{output_type}_mlir".format(
-            op = op,
```

Comment:
Maybe even include the {op} in these names

Reply:
included